pkgs/development/python-modules/submitit/default.nix

```diff
@@ -22,20 +22,18 @@ buildPythonPackage rec {
     hash = "sha256-uBlKbg1oKeUPcWzM9WxisGtpBu69eZyTetaANYpTG5E=";
   };
 
-  build-system = [
-    setuptools
-  ];
+  build-system = [ flit-core ];
 
   dependencies = [
     cloudpickle
-    flit-core
     typing-extensions
   ];
 
   nativeCheckInputs = [
     pytestCheckHook
     # event_loop was removed in pytest-asyncio 1.x
     pytest-asyncio_0
+    setuptools
```

Comment:
This belongs in `dependencies`:
https://github.com/facebookincubator/submitit/blob/1.5.3/submitit/core/plugins.py#L24
I opened https://github.com/facebookincubator/submitit/pull/1798.